quilt/tools/main.py

```diff
@@ -42,6 +42,8 @@ def main():
 
     push_p = subparsers.add_parser("push")
     push_p.add_argument("package", type=str, help="Owner/Package Name")
+    push_p.add_argument("--public", action="store_true",
+                        help="Create a public package (not valid if the package already exists")
```

Comment:
The help comment here is a bit confusing. How about, "Create or update a public package (fails if the package exists and is private)."